eos/rare-b-decays/bs-to-phi-ll.cc

```diff
@@ -130,7 +130,7 @@ namespace eos
             double beta = std::sqrt(beta2);
 
             // j1s
-            result[0] = 3.0 / 4.0 * (
+            result[0] = (
```

Comment:
As discussed, we would like to keep the current definition for the j's observables. I can take care of rearranging the factors of 3/4 once the rest is ready, if you want.